DNN Platform/DotNetNuke.Abstractions/Prompt/ICommand.cs

```diff
@@ -1,45 +1,16 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information
+
 namespace DotNetNuke.Abstractions.Prompt
 {
+    using System;
+
     /// <summary>
     /// This is used to retrieve and keep a list of all commands found in the installation.
     /// </summary>
-    public interface ICommand
+    [Obsolete("Deprecated in v9.9.1. Use IDnnCommand instead. Will be removed in v11.", false)]
+    public interface ICommand : IDnnCommand
```

Comment:
@mitchelsellers I think here everything was moved to IDnnCommand which ICommand inherits from, so basically it moves the code in the one we won't delete and the one that will be deleted is just basically a synomin or alias to the new one. I think this is non-breaking IMO.

Reply:
That might be true. But I believe the service locator will still be a concern possibly. I haven’t had the time to create a test usage to confirm.

Reply:
Yeah, I have not used that pattern much, I think the implementation is just simply right there in the file, but let's see what @bdukes thinks, and I'll make this noteworthy too to call out attention to testing in RC...

Reply:
It is a binary breaking change. If I built against `ICommand`, when I try to access one of the properties, it will fail, because I'm asking for `ICommand.Name`, for example, but that member no longer exists (even though there's a `Name` property on the class).

It is _not_ a _source_ breaking change. Recompiling would be all that's needed to support the newer interface. But you couldn't have one assembly that supported both versions.